src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String greeting;
+    private int localDate;
+    private int age;
+    private int size;
+
+    public static void main(String[] args) {
+        Dog dog = new Dog();
+        Cat cat = new Cat();
+    }
+
+    private String getFromOneToTen() {
+        String result = "";
+
+        for (int i = 0; i < 10; i++) {
+            if (i % 2 == 0) {
+                result = result + i + " ";
+            }
+        }
+        return result;
+    }
+
+    private void sayHello(String str) {
+        System.out.println("Hello, Mate! "
+                + "I am happy to see you! "
+                + "This class looks weird! Your task is to fix the checkstyle in this file. "
+                + "Your goal is to run `mvn clean package` and receive `BUILD SUCCESS` message");
+        System.out.println(str);
+    }
+
+    private void initializeVariables() {
+        greeting = "Hello mates!";
+        System.out.println(greeting);
+        localDate = LocalDate.now().getYear();
```

Comment:
To be absolutely precise, I would say it will be better to set instead of `localDate` variable name - just `year`, as far as `LocalDate.now().getYear()` returns the primitive int value for **year**.
https://docs.oracle.com/javase/10/docs/api/java/time/LocalDate.html#getYear()